features/step_definitions/helper_services.rb

```diff
@@ -161,6 +161,15 @@
     # So take the second one since this one can be implemented currently
     ###
     stats = {}
+    step %Q/I run the :label admin command with:/, table(%{
```

Comment:
I would suggest add a version check here as we only need this for 4.12+
A version check is much cheaper than adding a bunch of labels.